py/src/ai/h2o/sparkling/Initializer.py

```diff
@@ -43,7 +44,11 @@ class Initializer(object):
     @staticmethod
     def load_sparkling_jar():
         sc = SparkContext._active_spark_context
-        if Initializer.__sparklingWaterJarLoaded is not True and sc is not None:
+        if Initializer.__sparklingWaterJarLoaded is False and sc is None:
+            # Ensure that when we do import pysparkling, spark will put later the JAR file
+            # to the driver. This option has effect only when SparkContext has not been started before.
+            os.environ["PYSPARK_SUBMIT_ARGS"] = "--jars " + Initializer.__get_sw_jar() + " pyspark-shell"
```

Comment:
qq: Couldn't the original `os.environ["PYSPARK_SUBMIT_ARGS"]` contain other args `pyspark-shell`. Couldn't we override them with this statement?

Reply:
It might -> I can try to make it more robust and check for ``--jars`` presence